src/test/regress/citus_tests/config.py

```diff
@@ -321,11 +321,14 @@ def __init__(self, arguments):
         self.new_settings = {"citus.shard_replication_factor": 2}
         self.skip_tests = [
             # citus does not support foreign keys in distributed tables
-            # when citus.shard_replication_factor > 2
+            # when citus.shard_replication_factor >= 2
             "arbitrary_configs_truncate_partition_create", "arbitrary_configs_truncate_partition",
             # citus does not support modifying a partition when
-            # citus.shard_replication_factor > 2
-            "arbitrary_configs_truncate_cascade_create", "arbitrary_configs_truncate_cascade"]
+            # citus.shard_replication_factor >= 2
+            "arbitrary_configs_truncate_cascade_create", "arbitrary_configs_truncate_cascade",
+            # citus does not support colocating functions with distributed tables when
+            # citus.shard_replication_factor >= 2
+            "function_create", "functions"]
```

Comment:
I could consider splitting these files into two. Not all the tests there require we have `shard_replication_factor=1`

Reply:
It fails when we supply the distribution column argument.


```diff
 SELECT create_distributed_function('warning(int,text)','$1');
- create_distributed_function
----------------------------------------------------------------------
-
-(1 row)
-
+ERROR:  cannot colocate function "warning" and table "table_1"
```

The error message comes from https://github.com/citusdata/citus/blob/dd78c813781cd4ebc32bb049935a1a8f104693c4/src/backend/distributed/commands/function.c#L656-L668

Reply:
I think we can keep like this, no need to split. Thanks